spec/datadog/tracing/contrib/graphql/tracing_patcher_spec.rb

```diff
@@ -35,17 +35,5 @@
         end
       end
     end
-
-    context 'when given something else' do
-      before { remove_patch!(:graphql) }
-
-      it do
-        expect_any_instance_of(Datadog::Core::Logger).to receive(:warn).with(/Unable to patch/)
-
-        Datadog.configure do |c|
-          c.tracing.instrument :graphql, with_deprecated_tracer: true, schemas: [OpenStruct.new]
-        end
-      end
-    end
```

Comment:
I've removed this one GraphQL test because it adds little value (testing a warning), but to make it not flaky I need to split it out into it's own test file, Rake task, and execution group, because it pollutes the current application with a failed instrumentation.